src/app/(auth)/login/page.tsx

```diff
@@ -42,26 +46,29 @@ export default function Login() {
       return;
     }
 
-    const { error } = await auth.signIn(email, password);
+    setIsLoggingIn(true);
```

Comment:
was there any benefit to moving this up to before we call the `auth.signIn`?

Reply:
no clear benefits, but theoretically it should help setup a state change (which is async) before we await another async function, which should help parallelize the flow of execution